### PR TITLE
Integrate search suggestions

### PR DESCRIPTION
**Pull request by vileyne**

### CODE REVIEW

File: code/lightna/magento-os-frontend/js/common/SearchSuggestions.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            this.clearSuggestions();
+            return;
+        }
+
+        await this.getSuggestions();
+        await this.updateSuggestionsHtml();
+    }
+
+    async getSuggestions() {
+        this.suggestions = await Request.get(
```
Comment (vileyne) [inline code/lightna/magento-os-frontend/js/common/SearchSuggestions.js:34]: delay and abort?

Comment (vesan-83) [reply code/lightna/magento-os-frontend/js/common/SearchSuggestions.js:29]: delay + abort :)

Comment (vesan-83) [reply code/lightna/magento-os-frontend/js/common/SearchSuggestions.js:29]: Ideally delay + clearTimeout + abort

File: code/lightna/magento-os-frontend/css/components/common/search.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import "search-suggestions.css";
```
Comment (vesan-83) [inline code/lightna/magento-os-frontend/css/components/common/search.css:1]: will it stay here?

Comment (vileyne) [reply code/lightna/magento-os-frontend/css/components/common/search.css:1]: no, I will remove all imports in separate pr

Comment (vileyne) [reply code/lightna/magento-os-frontend/css/components/common/search.css:1]: thou makes no sense to introduce it here to remove later, will do it now

File: code/lightna/magento-os-frontend/js/common/Search.js [hunk 3]
```diff
@@ -1,24 +1,28 @@
...
     constructor() {
         this.component = $('.cjs-search');
-        this.search = $('#search');
-        this.searchWrap = $('.search__wrap');
-        this.clearAction = $(this.actions.clear);
-        this.searchAction = $(this.actions.search);
+        this.search = $(Search.selectors.input);
+        this.searchWrap = $(Search.selectors.wrapper);
+        this.clearAction = $(Search.actions.clear);
+        this.searchAction = $(Search.actions.search);
```
Comment (vesan-83) [inline code/lightna/magento-os-frontend/js/common/Search.js:25]: is Seanch. beter than this.?

**Review by vesan-83 — APPROVED** [review]

File: code/lightna/magento-os-frontend/js/common/Search.js (hunk 3 above)
Comment (vileyne) [reply code/lightna/magento-os-frontend/js/common/Search.js:25]: I wouldn't say so

**Pull request merged**